lib/fat_free_crm/mail_processor/base.rb

```diff
@@ -107,8 +107,8 @@ def with_new_emails
             end
           rescue Exception => e
             if %w[test development].include?(Rails.env)
```

Comment:
We should just use a proper logger and do 'debug' 'info' 'warning' or 'error' stuff here I think. Will raise a reminder for it.